### ISSUE TITLE
(Feature request) add an update button for desktop pet

### ISSUE THREAD

**Issue by deonsebastiandk** [Adrianotiger/desktopPet#86]
Hi,
Could somebody add an update button in desktop pet ? This could be very helpful as I notifies the new version of desktop pet

**Comment by Adrianotiger** [comment]
Yes, this would be a good idea. Maybe with an update notification on the tray icon.
